test/parallel/test-http-client-error-rawbytes.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const net = require('net');
+
+const response = Buffer.from('HTTP/1.1 200 OK\r\n' +
+  'Content-Length: 6\r\n' +
+  'Transfer-Encoding: Chunked\r\n' +
+  '\r\n' +
+  '6\r\nfoobar' +
+  '0\r\n');
+
+const server = net.createServer(common.mustCall((conn) => {
+  conn.write(response);
+}));
+
+server.listen(0, common.mustCall(() => {
+  const req = http.get(`http://localhost:${server.address().port}/`);
+  req.end();
+  req.on('error', common.mustCall((err) => {
+    const reason = 'Content-Length can\'t be present with chunked encoding';
```

Comment:
nit: backticks instead of single quotes to avoid escaping
```suggestion
    const reason = `Content-Length can't be present with chunked encoding`;
```

Reply:
As note: using backticks would not be accepted by eslint. Only double quotes would be accepted. Using backticks is only accepted by eslint if the template string contains a variable.